shelley/chain-and-ledger/formal-spec/chain.tex

```diff
@@ -830,6 +829,8 @@ \subsection{Operational Certificate Transition}
     \var{vk_{cold}}, the KES period number \var{n}, the KES period start \var{c_0} and the cold key
   signature.
 \item The hash \var{hk} of \var{vk_{cold}}.
+\item The slot \var{s} for the block.
+\item The number of KES periods that have elapsed on the certificate.
```

Comment:
I am not sure to correctly understand what "on the certificate" means here.

Reply:
ah. I think I can make that clearer by saying "The number of KES periods that have elapsed since the start period on the certificate."

Reply:
sounds a lot clearer, thanks!